st2actions/st2actions/handlers/mistral.py

```diff
@@ -31,12 +31,17 @@
 LOG = logging.getLogger(__name__)
 
 
-STATUS_MAP = dict()
-STATUS_MAP[action_constants.LIVEACTION_STATUS_REQUESTED] = 'RUNNING'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_SCHEDULED] = 'RUNNING'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_RUNNING] = 'RUNNING'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_SUCCEEDED] = 'SUCCESS'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_FAILED] = 'ERROR'
+STATUS_MAP = {
```

Comment:
Maybe we can add some code check here or in the tests to throw if not all of the statuses are mapped here.

This way issues like this will be detected earlier in the future.